python/ray/_private/function_manager.py

```diff
@@ -193,21 +192,28 @@ def fetch_and_register_remote_function(self, key):
                 function = pickle.loads(serialized_function)
             except Exception:
 
+                # If an exception was thrown when the remote function was
+                # imported, we record the traceback and notify the scheduler
+                # of the failure.
+                traceback_str = format_error_message(traceback.format_exc())
+
                 def f(*args, **kwargs):
                     raise RuntimeError(
-                        "This function was not imported properly.")
+                        "The remote function failed to import on the "
+                        "worker. This may be because needed library "
+                        "dependencies are not installed in the worker "
+                        "environment:\n\n{}".format(traceback_str))
```

Comment:
Is there anything actionable that we can tell the user? like, provide the dependency in a runtime environment? Maybe something like "One way to do that is to specify these dependencies in a runtime_env" and add a link to it?

Reply:
I'm unsure about this since runtime envs are an advanced feature. For most oss users, installing it via setup commands is likely the way to go.